### ISSUE TITLE
freezed claims for non-nullable Dart feature which is not exists yet

### ISSUE THREAD

**Issue by subzero911** [rrousselGit/freezed#248]
Flutter doctor:
```dart
$ flutter doctor -v
[√] Flutter (Channel stable, 1.20.1, on Microsoft Windows [Version 10.0.19041.388], locale ru-RU)
    • Flutter version 1.20.1 at D:\Flutter\Flutter SDK
    • Framework revision 2ae34518b8 (4 days ago), 2020-08-05 19:53:19 -0700
    • Engine revision c8e3b94853
    • Dart version 2.9.0

 
[√] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at D:\Android_SDK
    • Platform android-29, build-tools 29.0.3
    • Java binary at: D:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Android Studio (version 4.0)
    • Android Studio at D:\Program Files\Android\Android Studio
    • Flutter plugin version 46.0.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] Connected device (1 available)
    • AOSP on IA Emulator (mobile) • emulator-5554 • android-x86 • Android 9 (API 28) (emulator)
```

Error when I try to run a build_runner:
```dart
$ flutter pub run build_runner watch
Precompiling executable...
Precompiled build_runner:build_runner.
[INFO] Generating build script...
[INFO] Generating build script completed, took 429ms

[INFO] Creating build script snapshot......
[INFO] Creating build script snapshot... completed, took 14.0s

[INFO] Setting up file watchers...
[INFO] Setting up file watchers completed, took 41ms

[INFO] Waiting for all file watchers to be ready...
[INFO] Waiting for all file watchers to be ready completed, took 283ms

[INFO] Initializing inputs
[INFO] Building new asset graph...
[INFO] Building new asset graph completed, took 897ms

[INFO] Checking for unexpected pre-existing outputs....
[INFO] Checking for unexpected pre-existing outputs. completed, took 1ms

[INFO] Running build...
[INFO] Generating SDK summary...
[SEVERE] freezed:freezed on lib/generated/i18n.dart:

Bad state: Unexpected diagnostics:
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:119:41 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:152:17 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:88:62 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:153:38 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:186:51 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:133:32 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:154:25 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:18:17 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:64:4 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:168:32 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:47:14 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:159:38 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:132:37 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:118:48 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:19:11 - This requires the 'non-nullable' language feature to be enabled.
[SEVERE] freezed:freezed on lib/generated/i18n.dart:

Bad state: Unexpected diagnostics:
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:119:41 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:152:17 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:88:62 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:153:38 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:186:51 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:133:32 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:154:25 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:18:17 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:64:4 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:168:32 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:47:14 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:159:38 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:132:37 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:118:48 - This requires the 'non-nullable' language feature to be enabled.
D:\Flutter\Flutter SDK\bin\cache\pkg\sky_engine\lib\ui\channel_buffers.dart:19:11 - This requires the 'non-nullable' language feature to be enabled.
[INFO] 5.5s elapsed, 1/2 actions completed.
[INFO] 6.6s elapsed, 1/2 actions completed.
[INFO] 7.7s elapsed, 1/2 actions completed.
[INFO] 8.8s elapsed, 1/2 actions completed.
[INFO] 9.9s elapsed, 1/2 actions completed.
[INFO] 11.0s elapsed, 1/2 actions completed.
[INFO] 12.1s elapsed, 1/2 actions completed.
[INFO] 13.2s elapsed, 1/2 actions completed.
[INFO] 14.2s elapsed, 1/2 actions completed.
[INFO] 15.3s elapsed, 1/2 actions completed.
[INFO] 16.3s elapsed, 1/2 actions completed.
[INFO] 17.4s elapsed, 1/2 actions completed.
[INFO] 18.4s elapsed, 1/2 actions completed.
[INFO] 19.5s elapsed, 1/2 actions completed.
[WARNING] No actions completed for 15.0s, waiting on:
  - freezed:freezed on lib/main.dart
```

**Comment by rrousselGit** [comment]
Duplicate of #245
